screens/PointsHistoryScreen.js

```diff
@@ -0,0 +1,44 @@
+import * as Font from 'expo-font';
+import React from 'react';
+import { AsyncStorage, ScrollView, Text, View } from 'react-native';
+import getCustomerTransactions from './historyHelpers';
+
+import Transactions from '../components/Transactions';
+
+export default class PointsHistoryScreen extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      transactions: []
+    };
+  }
+
+  async componentDidMount() {
+    const userId = await AsyncStorage.getItem('userId');
+    getCustomerTransactions(userId).then(transactions => {
+      this.setState({ transactions });
```

Comment:
You'll want to split out the three most recent transactions into a separate array (three being arbitrary and as specified in Asana; maybe call it 'recents' and store that in state as well. That way you can easily display it later ⬇️

As always, refer to Figma for the wireframes